app/views/endemics/guidance/apply-guidance-for-farmers.njk

```diff
@@ -242,7 +242,7 @@
           <nav role="navigation" class="gem-c-related-navigation__nav-section govuk-!-margin-bottom-3" aria-labelledby="related-nav-related_items-da209313" data-module="gem-toggle" data-gem-toggle-module-started="true">
             <ul class="gem-c-contents-list__list govuk-list govuk-list--spaced govuk-!-margin-top-3">
               <li>
-                <a class="govuk-link"  href="https://fcp-ahwr-prototype.herokuapp.com/v26/farmer/guidance-visit">What happens on a livestock health and welfare review or endemic disease follow-up</a>
+                <a class="govuk-link"  href=" https://fcp-ahwr-prototype.herokuapp.com/v25/farmer/guidance-visit">What happens on a livestock health and welfare review or endemic disease follow-up</a>
```

Comment:
Why those are prototype links? just use #, if they are not confirmed yet

Reply:
> Why those are prototype links? just use #, if they are not confirmed yet

done

Reply:
> Why those are prototype links? just use #, if they are not confirmed yet

done